### PR TITLE
[widgets] Add boolean content checker to text fields

### PR DESCRIPTION
**Pull request by adsteel**
Addressing issue https://github.com/mojotech/dill/issues/31

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -186,13 +186,21 @@ If you'd rather use a CSS selector, you can do that by passing the second argume
...
   widget(:form_with_everything).state #=> 'CO'
 ```
 
+For text fields, you can check if there is any content in the field by calling the field method + question mark.
+
+```ruby
+  # <input type="text" value="">
+  widget(:form_with_everything).request? # => false
+  # <input type="text" value="More info">
+  widget(:form_with_everything).request? # => true
```
Comment (joelhawksley) [inline docs/getting_started.md]: For the sake of clarity, might we continue the `state` example in these steps?

**Comment by adsteel** [comment]
@joelhawksley Updated the readme.

**Comment by joelhawksley** [comment]
:+1:

**Comment by aesnyder** [comment]
awesome :tada: :+1:

**Pull request merged**